.travis.yml

```diff
@@ -162,6 +172,7 @@ matrix:
         - libgl1-mesa-dev
         - libegl1-mesa-dev
         - libxrandr-dev
+        - gperf
```

Comment:
I do not think it is needed.

Reply:
fontconfig build dependency in Linux and Windows